src/components/hint/index.tsx

```diff
@@ -251,12 +302,12 @@ class Hint extends BaseComponent {
   }
 
   getHintPadding() {
-    const paddings = {paddingVertical: this.hintOffset, paddingHorizontal: this.edgeMargins};
-    if (this.useSideTip) {
+    const paddings: Paddings = {paddingVertical: this.hintOffset, paddingHorizontal: this.edgeMargins};
+    if (this.useSideTip && this.targetLayout?.x) {
```

Comment:
Are you sure this does not break anything, when not setting the `paddings.paddingLeft` to undefined?

Reply:
Not sure I understand the issue. `this.targetLayout.x` is necessary when setting both `paddings.paddingLeft` and `paddings.paddingRight`. This prevents a runtime error if `this.targetLayout` is undefined. It seems to work fine as much as I tested. Am I missing something?